courses/machine_learning/deepdive2/structured/solutions/5b_deploy_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>remove last paragraph</p><p> </p><p>Also (personal preference) first paragraph seems redundant. The second paragraph says it all well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Same as above.</p>